PsychSourceGL/Source/Common/Base/PythonGlue/PsychScriptingGluePython.c

```diff
@@ -54,14 +54,25 @@
 // in February 2013:
 #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
 #include <numpy/arrayobject.h>
-// Define after include, to avoid compiler warning and pass runtime loader compat check:
-#undef NPY_FEATURE_VERSION
-#define NPY_FEATURE_VERSION NPY_1_7_API_VERSION
-
-// Can not use NPY_TITLE_KEY macro if compat limited api is selected.
-// However, i have no clue what we'd use it for, so there...
-#ifdef Py_LIMITED_API
-#undef NPY_TITLE_KEY
+
+#if defined(Py_LIMITED_API) && defined(__GNUC__) && (__GNUC__ < 10)
```

Comment:
This workaround won't fly. This is a literal copy of code that is GPL-3+ licensed, and I don't want GPL code in core components. This would effectively change the license of all the PTB python extension modules from MIT to GPL-3, a no-go.

One really doesn't want GPL license for anything that might have to interface/link with proprietary 3rd party (aka "non-system") libraries, like not uncommon for proprietary research hardware, e.g., many eye trackers etc.

Better to drop Py_LIMITED_API support for gcc < 10, using suitable checks in setup.py or a #error abort here with advise to upgrade to gcc-10+. Ubuntu 20.04 ships gcc < 10 by default, but also has a gcc-10 package, so at least that one can still build with gcc-10 installed.

Reply:
Sounds good (and sorry for introducing incompatibly-licensed code), I'll `#error` there.